bindings/java/iota-wallet-java/examples/ExampleProject/src/ExampleUtils.java

```diff
@@ -5,14 +5,19 @@
 import org.iota.types.AccountHandle;
 import org.iota.types.account_methods.SyncAccount;
 import org.iota.types.exceptions.WalletException;
+
 public class ExampleUtils {
 
     public static AccountHandle setUpAccountWithFunds(Wallet wallet, String alias) throws WalletException, InterruptedException {
         AccountHandle a = wallet.createAccount(alias);
-        while(a.syncAccount(new SyncAccount()).getBaseCoin().getAvailable().equals("0")) {
+
+        int available = Integer.parseInt(a.syncAccount(new SyncAccount()).getBaseCoin().getAvailable());
+
+        while(available < 10000000) {
```

Comment:
Maybe `!= 0` instead?

Reply:
But lets say the balance is 0.5 SMR then it won't ask for funding ?
And with the 0.5 SMR i can't do much because of the storage deposit limits

Reply:
Okay makes sense if you want to do more than the minimum